steps/cadence-innovus-signoff/configure.yml

```diff
@@ -39,15 +40,16 @@ commands:
   # Outputs
   - cd outputs
   - ln -sf ../checkpoints/design.checkpoint
-  - ln -sf ../typical.spef.gz      design.spef.gz
-  - ln -sf ../results/*.gds.gz     design.gds.gz
-  - ln -sf ../results/*.lvs.v      design.lvs.v
-  - ln -sf ../results/*.vcs.v      design.vcs.v
-  - ln -sf ../results/*.vcs.pg.v   design.vcs.pg.v
-  - ln -sf ../results/*.lef        design.lef
-  - ln -sf ../results/*.pt.sdc     design.pt.sdc
-  - ln -sf ../results/*.sdf        design.sdf
-  - ln -sf ../results/*.virtuoso.v design.virtuoso.v
+  - ln -sf ../typical.spef.gz             design.spef.gz
+  - ln -sf ../results/*.gds.gz            design.gds.gz
+  - ln -sf ../results/*innovus_merged.gds design_innovus_merged.gds
```

Comment:
Very minor, but can we make it "design.merged.gds", just matching the naming convention of the others.

Reply:
Is it gds or gds.gz?

Reply:
I intentionally did not do that to avoid potentially breaking existing flows. There could be ambiguity on which design_merged.gds is meant to be connected to downstream nodes.

Reply:
Also it's gds. The existing Calibre drc/lvs nodes expect a .gds, so making it a .gds.gz causes them to error out.

Reply:
I think it should be either "design.merged.gds" or "design-merged.gds".

Just a preference for not using underscores. Also I still don't like putting "innovus" in the filename. There will be ambiguity either way about which merged gds to use. There are multiple design.v in the flow too after all. The intention of the spec I think is that if there is any question about what a file is, it should be documented in a comment in the configure.yml right there.

If possible, I would rename...

Reply:
ok it's changed to design-merged now